packages/shared/components/organisms/AccountActivity.svelte

```diff
@@ -1,23 +1,15 @@
 <script lang="typescript">
-    import { ActivityRow, TogglableButton, Text, IconTextInput } from 'shared/components'
+    import { ActivityRow, TogglableButton, Text, TextInput } from 'shared/components'
     import { localize } from '@core/i18n'
     import { openPopup } from 'shared/lib/popup'
-    import {
-        isSyncing,
-        getIncomingFlag,
-        isFirstSessionSync,
-        walletSetupType,
-        getAccountMessages,
-    } from 'shared/lib/wallet'
+    import { isSyncing, isFirstSessionSync, walletSetupType } from 'shared/lib/wallet'
     import { SetupType } from 'shared/lib/typings/setup'
-    import { AccountMessage } from 'shared/lib/typings/wallet'
+    import { FontWeightText } from '../Text.svelte'
```

Comment:
We can use `import { FontWeightText } from 'shared/components/Text.svelte'` to avoid relative paths :+1: